### PR TITLE
Scrutinizer Auto-Fixes

### PR DESCRIPTION
**Pull request by scrutinizer-auto-fixer**
@DeepDiver1975 requested this pull request.

It consists of patches automatically generated for this project on Scrutinizer:
https://scrutinizer-ci.com/g/owncloud/core/

### CODE REVIEW

File: lib/private/api.php [hunk 1]
```diff
@@ -329,7 +329,7 @@ private static function isAuthorised($action) {
 
 	/**
 	 * http basic auth
-	 * @return string|false (username, or false on failure)
+	 * @return string (username, or false on failure)
```
Comment (MorrisJobke) [inline lib/private/api.php]: That's wrong - see line 336

File: lib/private/group/manager.php [hunk 2]
```diff
@@ -150,6 +150,9 @@ public function get($gid) {
 		return $this->getGroupObject($gid);
 	}
 
+	/**
+	 * @param string $gid
+	 */
```
Comment (nickvergessen) [inline lib/private/group/manager.php]: `@return IGroup`

File: lib/private/preview.php [hunk 3]
```diff
@@ -921,9 +921,9 @@ private function resizeAndStore($fileId) {
 	 * @param int $askedWidth
 	 * @param int $askedHeight
 	 * @param int $previewWidth
-	 * @param null $previewHeight
+	 * @param integer $previewHeight
 	 *
-	 * @return int[]
+	 * @return double[]
```
Comment (nickvergessen) [inline lib/private/preview.php]: we do `round()` so int is correct?

File: lib/private/preview.php [hunk 4]
```diff
@@ -971,7 +971,7 @@ private function scale($image, $askedWidth, $askedHeight, $previewWidth, $previe
 	 * @param int $askedWidth
 	 * @param int $askedHeight
 	 * @param int $previewWidth
-	 * @param null $previewHeight
+	 * @param double $previewHeight
```
Comment (nickvergessen) [inline lib/private/preview.php]: this is int too

File: lib/private/preview.php [hunk 5]
```diff
@@ -990,7 +990,7 @@ private function crop($image, $askedWidth, $askedHeight, $previewWidth, $preview
...
```
Comment (nickvergessen) [inline lib/private/preview.php]: int

File: lib/public/appframework/utility/icontrollermethodreflector.php [hunk 6]
```diff
@@ -36,6 +36,7 @@
 	 * @param object $object an object or classname
 	 * @param string $method the method which we want to inspect
 	 * @since 8.0.0
+	 * @return void
```
Comment (nickvergessen) [inline lib/public/appframework/utility/icontrollermethodreflector.php]: return before since?

File: lib/public/search/pagedprovider.php [hunk 7]
```diff
@@ -59,6 +59,7 @@ public function search($query) {
 	 * @param string $query
 	 * @param int $page pages start at page 1
 	 * @param int $size, 0 = SIZE_ALL
+	 * @param integer $size
```
Comment (nickvergessen) [inline lib/public/search/pagedprovider.php]: duplicates the line above, it just has a "typo" on the name, (the comma)

**Comment by nickvergessen** [comment]
Fixed my comments, :+1: for the remaining stuff

File: lib/private/group/dummy.php [hunk 8]
```diff
@@ -106,7 +106,7 @@ public function addToGroup($uid, $gid) {
 	 * Removes a user from a group
 	 * @param string $uid Name of the user to remove from group
 	 * @param string $gid Name of the group from which remove the user
-	 * @return bool
+	 * @return null|false
```
Comment (rullzer) [inline lib/private/group/dummy.php]: No this should be bool...
We actually should have proper interfaces for the backends...

And what is the dummy group backend doing in lib.... should that not be in test?

**Comment by rullzer** [comment]
Dummy_Group backend now properly returns a bool.
:+1: for the rest.

**Pull request merged**